project/OuTingYun/README.md

```diff
@@ -0,0 +1,95 @@
+# Image Filtering 
+
+## Basic Information
+One common way to implement image filtering is using a NxN kernel to convolute the origin image. By setting different value in kernel, you will obtain distinct image after convoluting.The other way is to calculate the rgb separately according to the characteristics of the filter, for example, We get the new value by subtracting the old value from 255 for Film filter.
+
+## Problem to Solve
+
+In this project, I use c++ to make the calculation of image filters faster. The use python to show and store image. 
+
+**Gussian blur** is useful for bluring image by Guassian function. It is widely use in Adobe Photoshop, GIMP, Paint.NET and so on. We setting the kernel with the normal distribution  in 2-dimension space: 
```

Comment:
Gaussian blur is a straight-forward algorithm to implement and that makes is a good topic.